syntax = "proto3";

option go_package= "project1/hello_grpc";
package hello_grpc;

service MyService {
    rpc MyFunction(MyNumber) returns (MyNumber) {}
}

message MyNumber {
    int32 value = 1;
}
